lib/codebreaker/processor.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'interface.rb'
+# comment
+class Processor
+  def initialize
+    @attempts = 3
+  end
+
+  include Interface
+  def turn_processor(code, guess)
```

Comment:
This method makes two actions:
1. print messages
2. return `results_array`
You need to do one action in on method. Please divide logic.